transform/binary-plugin/examples/test/annotation/annotation.go

```diff
@@ -10,12 +10,7 @@ import (
 )
 
 func main() {
-	u, err := cli.Unstructured(cli.ObjectReaderOrDie())
-	if err != nil {
-		cli.WriterErrorAndExit(fmt.Errorf("error getting unstructured object: %#v", err))
-	}
-
-	cli.RunAndExit(cli.NewCustomPlugin("AnnotationPlugin", Run), u)
+	cli.RunAndExit(cli.NewCustomPlugin("WhiteoutPlugin", "v1", nil, Run))
```

Comment:
Possible bad merge with my plugin name fix. Should preserve the name "AnnotationPlugin"